src/main/tables.ts

```diff
@@ -198,7 +200,15 @@ export async function importCSV(
 
       await sql`GRANT SELECT ON ${sql(tableName)} TO ${sql(postgresConfig.trustedUser)}`;
     });
+
+    if (isOverwriting) {
+      // Examples might have become stale now; remove them and they will be rebuilt on first view.
+      // Keep this async to not crash the import in case of failure.
+      removeTableExamples(tableName);
+    }
```

Comment:
Maybe drop the `isOverwriting` flag in case we have unsynchronized garbage and always try to remove them.

I'd avoid the fire and forget promise in favor of a `try/catch`:

```
try {
  await removeTableExamples(tableName);
} catch (e) {
  console...
}
```